content/news.md

```diff
@@ -4,6 +4,48 @@ title: News
 
 > The [press](/press) page contains a full listing of news coverage.
 
+**October 30, 2020**
+
+The Python Software Foundation live-streams a [key generation and signing ceremony](https://www.youtube.com/watch?v=jjAq7S49eow&t=3078s) that marks the first practical steps in deploying The Update Framework (TUF) to the [Python Package Index](pypi.org).  
+
+**February 15, 2020**
+
+PEP 458, Secure PyPI Downloads with Package Signing, is accepted and merged into the Python master.
```

Comment:
```suggestion
[PEP 458](https://www.python.org/dev/peps/pep-0458/), Secure PyPI Downloads with Package Signing, is accepted and merged into the Python master.
```

Reply:
Thanks, Marina. I wasn't sure what was considered the official "document."